packages/images/lib/compare.js

```diff
@@ -43,15 +43,30 @@ async function compareImages (mode, firstImage, secondImage, options = {}) {
     case MATCH_TEMPLATE_MODE.toLowerCase():
       // firstImage/img1 is the full image and secondImage/img2 is the partial one
       result = await imageUtil.getImageOccurrence(img1, img2, options);
+      if (options.multiple) {
+        return result.multiple.map(convertVisualizationToBase64);
+      }
       break;
     default:
       throw new errors.InvalidArgumentError(`'${mode}' images comparison mode is unknown. ` +
         `Only ${JSON.stringify([MATCH_FEATURES_MODE, GET_SIMILARITY_MODE, MATCH_TEMPLATE_MODE])} modes are supported.`);
   }
-  if (!_.isEmpty(result.visualization)) {
-    result.visualization = result.visualization.toString('base64');
+  return convertVisualizationToBase64(result);
+}
+
+/**
+ * base64 encodes the visualization part of the result
+ * (if necessary)
+ *
+ * @param {OccurenceResult} element - occurrence result
+ *
+**/
+function convertVisualizationToBase64 (element) {
+  if (!_.isEmpty(element.visualization)) {
```

Comment:
`isString` was wrong since the visualization was not string in this case...
Finally, the failure was resolved. Wrapping up this PR
https://github.com/appium/appium-plugins/runs/2636661778?check_suite_focus=true